### ISSUE TITLE
Changes coming to tidyselect v1.2.0

### ISSUE THREAD

**Issue by ddsjoberg** [larmarange/broom.helpers#173]
Hey hey!

FYI, there are changes coming to tidyselect that affect broom.helpers. They are making changes to the way that the selectors can be used and deprecating some of the ways we used it here. I don't know/understand all the details yet. But I do know that there are errors in broom.helpers during the reverse dependency checks (see https://github.com/r-lib/tidyselect/issues/312).

The tidyselect team will likely reach out soon, but we can start looking into this earlier if there is time.

**Comment by larmarange** [comment]
Thanks for flagging it. We need to better understand what are the tidyselect teams recommendations.

We use `.data$var` quite a lot and it seems that we will have to switch to `all_of("var")`.

We need to see if there are other issues.

**Comment by larmarange** [comment]
> We use `.data$var` quite a lot and it seems that we will have to switch to `all_of("var")`.

We need to see if it is just `dplyr::select()` which is affected but not other dplyr verbs like `mutate()`.

**Comment by hadley** [comment]
I think we’ll make a change to tidyselect so broom.helpers doesn’t fail (https://github.com/r-lib/tidyselect/pull/313), which is why we haven’t reached out yet.

**Comment by ddsjoberg** [comment]
Thank you @hadley ! I was getting nervous about getting these updates quickly completed in broom.helpers, then moving on to the rev deps of broom.helpers before the tidyselect release. The extra time to address the update is valuable for us!

@larmarange I started prepping a PR to address the changes in tidyselect and ran the GH Actions against the dev versions of tidyselect and tidyr. Those updates have now been addressed, but we are seeing a new failure due to emmeans: they now have a requirement of R v4.1 or higher, so just need to update an example
